### ISSUE TITLE
Multi-field join

### ISSUE THREAD

**Issue by tasso85** [atk4/dsql#115]
The library currently doesn't seem to support multi field joins, or is there already a way to perform them?

I was thinging something like

```
SELECT A.*
FROM my_tbl_a A
JOIN my_tbl_b B ON (A.x = B.y AND A.b = B.b)
WHERE ...
```

**Comment by romaninsh** [comment]
I believe you can use join(table, expression) but I must done check presence of doc

**Comment by tasso85** [comment]
I must have missed that completely in the documentation.

Sorry for the false issue reported, it works indeed!

**Comment by romaninsh** [comment]
I'll keep the ticket as reminder to add doc

**Comment by romaninsh** [comment]
Resolved in 57cb864, but i mistakingly specified issue #116
